libyul/optimiser/FullInliner.cpp

```diff
@@ -80,10 +80,20 @@ void FullInliner::run()
 
 	// TODO it might be good to determine a visiting order:
```

Comment:
Isn't this +- what this PR does now?

Reply:
This PR makes it less chaotic, but the TODO suggests that we should find an order that results in better code.